python/py-igraph/Portfile

```diff
@@ -4,40 +4,39 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                py-igraph
-version             0.7
-categories-append   math
+version             0.8.0
+categories-append   math science
 platforms           darwin
 license             GPL-2+
 
-python.versions     27 35 36
+python.versions     27 35 36 37 38
 
 maintainers         {snc @nerdling} openmaintainer
 
-description         Python interface to the igraph library
+description         Python interface to the igraph library.
 
-long_description    Python interface to the igraph high performance \
-                    graph library
+long_description    Python interface to the igraph library for network analysis \
+                    and graph theory. To enable visualization functionality, \
+                    it is recommended to also install py-cairocffi.
 
 homepage            https://pypi.python.org/pypi/python-igraph/
 master_sites        pypi:p/python-igraph/
 
 distname            python-igraph-${version}
 dist_subdir         python/${distname}/1/
```

Comment:
this shouldn't be necessary... or?

Reply:
I don't know. It was like this. @nerdling ?

Reply:
this is likely a left-over from a previous [stealth update](https://trac.macports.org/wiki/PortfileRecipes#stealth-updates). It builds fine without this line and should be removed.

Reply:
It's already removed, so marking this as resolved.